ci/docker/docker_push.sh

```diff
@@ -0,0 +1,8 @@
+set -e
+set +x
```

Comment:
I recommend a comment similar to https://github.com/envoyproxy/envoy/blob/master/ci/docker_push.sh#L3 here.

Reply:
Update this to work mostly like Envoy's version, thanks for pointing that out